packages/salesforcedx-apex-replay-debugger/test/unit/commands/apexExecutionOverlayResultCommand.test.ts

```diff
@@ -684,8 +684,8 @@ export function createExpectedXHROptions(
       Accept: 'application/json',
       Authorization: `OAuth 123`,
       'Content-Length': requestBody
-        ? Buffer.byteLength(requestBody, 'utf-8')
-        : 0,
+        ? String(Buffer.byteLength(requestBody, 'utf-8'))
+        : '0',
```

Comment:
Headers in http land can only be strings.  They tightened this up on the types here. I verified that things still work as expected for toggling/clearing checkpoints where is one of the only places we actually use the requestService.

Reply:
Sweet.  Thx for converting these all to strings.